Modules/Sounds.lua

```diff
@@ -44,9 +44,11 @@ function Sounds.PlayQuestComplete()
         return
     end
 
-    shouldPlayObjectiveSound = false
     shouldPlayObjectiveProgress = false
-    PlaySoundFile(Sounds.GetSelectedSoundFile(Questie.db.profile.questCompleteSoundChoiceName), "Master")
+    shouldPlayObjectiveComplete = false
+    C_Timer.After(Questie.db.profile.soundDelay, function ()
+        PlaySoundFile(Sounds.GetSelectedSoundFile(Questie.db.profile.questCompleteSoundChoiceName), "Master")
+    end)
```

Comment:
We don't want a delay for this. As soon as Questie knows the quest is complete, play the sound.